packages/golden_toolkit/test/sample_widgets.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class WidgetWithShadows extends StatelessWidget {
```

Comment:
just a widgets with drop shadows, useful to test that indeed shadows are drawn properly or not